aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/sampler/test_aws_xray_remote_sampler.py

```diff
@@ -57,6 +57,10 @@ def test_create_remote_sampler_with_empty_resource(self):
         self.assertIsNotNone(rs._AwsXRayRemoteSampler__resource)
         self.assertTrue(len(rs._AwsXRayRemoteSampler__client_id), 24)
 
+        # Clean up timers
+        rs._rules_timer.cancel()
+        rs._targets_timer.cancel()
+
```

Comment:
Can we add a `setUp` and `tearDown` methods to this class to ensure we do not add flaky tests in future? Can make `rs` a class variable I think?

Reply:
Added your suggestion in latest commit